stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/StreamFileProvider.kt

```diff
@@ -20,5 +23,22 @@ public class StreamFileProvider : FileProvider() {
 
         public fun getUriForFile(context: Context, file: File): Uri =
             getUriForFile(context, getFileProviderAuthority(context), file)
+
+        public fun writeImageToSharableFile(context: Context, bitmap: Bitmap): Uri? {
```

Comment:
Moved this method here as it's related to file management and doesn't depend on our image loading library